tests/bruno/e2e/cross-chain-hub/13-node1-send-shutdown-channel.bru

```diff
@@ -41,5 +41,5 @@ assert {
 
 script:post-response {
   // Sleep for sometime to make sure current operation finishes before next request starts.
-  await new Promise(r => setTimeout(r, 100));
+  await new Promise(r => setTimeout(r, 1000));
```

Comment:
I guess it is not a timing issue, this step failed with 500:
```
e2e/cross-chain-hub/17-node1-close-ln-channel (500 Internal Server Error) - 3 ms
```

Reply:
yes, I found the root cause: the shutdown script is different with the one when building commitment tx, the latest commit should resolve it.